src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/CustomerRegistrationFormSubscriber.php

```diff
@@ -33,29 +32,26 @@ public static function getSubscribedEvents(): array
         ];
     }
 
-    /**
-     * @throws \InvalidArgumentException
-     */
     public function preSubmit(FormEvent $event): void
     {
         $rawData = $event->getData();
         $form = $event->getForm();
         $data = $form->getData();
 
-        Assert::isInstanceOf($data, CustomerInterface::class);
+        if ($data instanceof CustomerInterface) {
```

Comment:
Would be more readable if inverted.
```suggestion
        if (!$data instanceof CustomerInterface) {
            return;
        }
        ...
```